docs/connecting-data/connecting-to-apis-and-external-data/virustotal-api.md

```diff
@@ -0,0 +1,9 @@
+---
+description: How to Connect DataDistillr to the VirusTotal API
```

Comment:
Also add VirusTotal to the readme.md file within connecting-to-apis-and-external-data